lib/linguist/languages.yml

```diff
@@ -1475,8 +1475,9 @@ IGOR Pro:
 INI:
   type: data
   extensions:
-  - .ini
   - .cfg
+  - .ini
```

Comment:
The first file extension is considered the primary extension. Only the subsequent extensions need to be sorted. Thus, `.ini` should remain in first position ;)